wanda/as_filter/as_filter.py

```diff
@@ -27,14 +29,14 @@ def get_prefix_lists(self):
             v4_set.update(result_entries_v4_cleaned)
             v6_set.update(result_entries_v6_cleaned)
 
-        if len(v4_set) == 0 and len(v6_set) == 0:
+        if len(v4_set) == 0 and len(v6_set) == 0 and self.is_customer:
             l.error(f"{self.autos} has no v4 filter lists.")
```

Comment:
I fucked up the error message while moving this to `as_filter.py`, may you correct the message AND the exception?